bedrock/firefox/templates/firefox/features/index.html

```diff
@@ -78,6 +78,9 @@ <h2 class="section-title"><span>{{ _('Modern') }}</span></h2>
           </a>
         </p>
       </li>
+    {% set modern_iterator = 4 %}
+    {% if l10n_has_tag('firefox-features-send-tabs') %}
+      {% set modern_iterator = 5 %}
```

Comment:
While this works and preserves the data attributes as they are in the current form, my gut says this is only prolonging the existence of the same issue for a bit longer. What happens when there's another tag for example, would just become more complicated again? While I'm not opposed to merging it this way, my preference would be to either:

- Remove this data attribute altogether (how useful really is it?)
- Rework the feature list items into a macro, which could generate this number dynamically (this would likely be a fair bit of work, so I wouldn't expect it to happen here).
- Find some alternative solution for GA that doesn't rely so much on a specific number.